xml/System.Text.Json/JsonElement.xml

```diff
@@ -170,9 +170,22 @@ This method does not parse the contents of a JSON string value.
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Gets the value of the element as a byte array.</summary>
+        <returns>The value decoded as a byte array.</returns>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+
+## Remarks
+
+This method does not create a byte representation of values other than base 64-encoded JSON strings.
```

Comment:
```suggestion
This method does not create a byte[] representation of values other than base 64-encoded JSON strings.
```

Reply:
I slightly modified your suggestion: Needs to be wrapped by back tick characters.